ui/ui-components/pages/property/[id]/edit.tsx

```diff
@@ -27,6 +30,15 @@ export default function Page(props) {
     filterOptions,
     properties,
     hydrationError,
+  }: {
+    errorHandler: ErrorHandler;
+    successHandler: SuccessHandler;
+    sources: Models.SourceType[];
+    propertiesHandler: PropertiesHandler;
+    types: Actions.AppOptions["types"];
+    filterOptions: Actions.ScheduleFilterOptions["options"];
+    properties: Models.PropertyType[];
+    hydrationError: Error;
```

Comment:
Here's an example where we're referencing the `Error` type... but we use the same terminology/variable name.